railties/lib/rails/test_unit/testing.rake

```diff
@@ -23,13 +23,14 @@ namespace :test do
 
   task run: %w[test]
 
-  desc "Run tests quickly, but also reset db"
+  desc "Runs tests quickly, but also resets the database"
   task :db do
     success = system({ "RAILS_ENV" => ENV.fetch("RAILS_ENV", "test") }, "rake", "db:test:prepare", "test")
     success || exit(false)
   end
 
   Rails::TestUnit::Runner::TEST_FOLDERS.each do |name|
+    desc "Runs tests in test/#{name}"
```

Comment:
I see that the output of `bin/rails --help` mixes different verb forms, but the common commands all use the infinitive form:

```
You must specify a command. The most common commands are:

  generate     Generate new code (short-cut alias: "g")
  console      Start the Rails console (short-cut alias: "c")
  server       Start the Rails server (short-cut alias: "s")
  test         Run tests except system tests (short-cut alias: "t")
  test:system  Run system tests
  dbconsole    Start a console for the database specified in config/database.yml
               (short-cut alias: "db")
  plugin new   Create a new Rails railtie or engine
```

So should we use that for these?

```suggestion
    desc "Run tests in test/#{name}"
```

That would also save one character of precious screen space! :wink: